_rules/html-page-lang-valid-bf051a.md

```diff
@@ -63,31 +63,31 @@ The `lang` attribute specified is neither empty ("") nor only [ASCII whitespace]
 
 #### Failed Example 1
 
-The `lang` attribute value is not a valid primary language subtag.
+This `html` element has a `lang` attribute whose value is not a [valid language subtag][].
 
 ```html
 <html lang="xyz"></html>
 ```
 
 #### Failed Example 2
 
-The `lang` attribute value has a valid primary language subtag, but a syntactically invalid region subtag.
+This `html` element has a `lang` attribute whose value is not a [valid language subtag][]. Even though the [primary language subtag][] is valid, the [region subtag] is not
 
 ```html
 <html lang="en-US-GB"></html>
 ```
 
 #### Failed Example 3
 
-The `lang` attribute value is not a valid primary language subtag.
+This `html` element has a `lang` attribute whose value is not a [valid language subtag][].
 
 ```html
 <html lang="123"></html>
 ```
 
 #### Failed Example 4
```

Comment:
Does this really need 3 examples that are not valid subtags?

Reply:
I agree that Failed Example 1 and 3 are the same.
Subject for some other PR…